source/public/index.html

```diff
@@ -45,28 +53,40 @@ <h1>Welcome back to your developer journal!</h1>
         </section>
 
         <section id="projects">
-            <h2>Ongoing Projects</h2>
+            <h2>Projects</h2>
+                 <button class="newProjectButton" title="new project">
+                    + New Project
```

Comment:
Could you replace the `+` here with the font-awesome icon?
`<i class="fa-solid fa-plus"></i>`